### PR TITLE
US 1583733 Add missing language IDs - 06

### PR DESCRIPTION
**Pull request by v-thpra**
US 1583733 - Fix CATS P1 issues by adding missing language IDs to code blocks.
- Used "output" for program output examples: Some of these could be XML but they include some plain text output also, so they are not strictly XML.

Contributes to #2192

### CODE REVIEW

File: docs/csharp/language-reference/tokens/interpolated.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ The structure of an item with an interpolation expression is as follows:
 {<interpolationExpression>[,<alignment>][:<formatString>]}
 ```
 
-Elements in square brackets are optional. The following table describes each element:
+Elements in square brackets are optional. The following table describes each element:
```
Comment (pkulikov) [inline docs/csharp/language-reference/tokens/interpolated.md]: I guess that was a typo to add unnecessary white spaces:
```suggestion
Elements in square brackets are optional. The following table describes each element:
```

**Comment by v-thpra** [comment]
@mairaw Some of these could be XML but they include some plain text output also, so they are not strictly XML. If you want them changed just let me know. Thanks!

File: docs/csharp/language-reference/tokens/interpolated.md (hunk 1 above)
Comment (v-thpra) [reply docs/csharp/language-reference/tokens/interpolated.md]: Good catch Petr. I deleted the extra spaces.

**Review by BillWagner — APPROVED** [review]
Thanks for making these updates @v-thpra 

I'll :shipit: now.

**Pull request merged**